CubicrootXYZ/matrix-reminder-and-calendar-bot
# allow to add multiple users per room

## [ENHANCEMENT]: allow to add multiple users per room

Adds a new action that adds a user to the bot so multiple users in a room can interact with the bot.

**Related Issues:**

- #51 

### Testing

1. Add a user to a channel with the bot
2. write `add user @exampleuser:matrix.org`
3. Check if this user can now interact with the bot too

### Dependencies

none

### Before merge checklist

- [x] Changes are tested (code is actually executed and proofed working)
- [x] Documentation reflects the changes if necessary
- [x] Documentation states the newest version number
- [x] Pipeline/Action is executed successfully
- [x] All `Todo`s in the code done

### After merge checklist (for code owners)

- [ ] Version number increased if necessary 
- [ ] Version number increased in documentation if necessary
- [ ] Changes added to draft of next release
